src/test/scala/uk/gov/nationalarchives/tdr/api/routes/FileMetadataRouteSpec.scala

```diff
@@ -113,19 +112,19 @@ class FileMetadataRouteSpec extends AnyFlatSpec with Matchers with TestRequest w
   }
 
   private def checkFileMetadataExists(fileId: UUID): Unit = {
-    val sql = "select * from FileMetadata where FileId = ?;"
+    val sql = "select * from FileMetadata where FileId = ? AND PropertyName = ?;"
```

Comment:
Upper case commands for consistency?

Reply:
I've changed this one and the other ones in this file to upper case.